angular/projects/spark-angular/src/lib/components/sprk-modal/sprk-modal.component.ts

```diff
@@ -51,7 +51,7 @@ import * as _ from 'lodash';
         >
           <div
             *ngIf="modalType == 'wait'"
-            class="sprk-o-Stack__item sprk-c-Spinner sprk-c-Spinner--circle sprk-c-Spinner--large sprk-c-Spinner--dark"
+            class="sprk-o-Stack__item sprk-c-Spinner sprk-c-Spinner--circle sprk-c-Spinner--large sprk-c-Spinner--red"
```

Comment:
We avoid color specific css names in cases like this when need to change out our colors. Whats wrong with dark? Red is still the dark version thats meant for lighter backgrounds and in keeping it, we avoid the breaking change.

Reply:
We still need a "dark" version for the tertiary buttons, which is black. We can come up with a different name, for sure. But we need to have the light version, the black version and the red version.